rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py

```diff
@@ -137,14 +144,8 @@ def list_files(package, subdir, file_extension='.launch'):
         @param file_extension: Defaults to '.launch', ``str``
         :returns: list of msgs/srv in package, ``[str]``
         """
-        if subdir == None or subdir == '':
-            raise ValueError('Invalid package subdir = {}'.format(subdir))
-
-        rospack = rospkg.RosPack()
-
-        path = os.path.join(rospack.get_path(package), subdir)
-
-        return [genmsg.resource_name(package, t) for t in RqtRoscommUtil._list_types(path, file_extension)]
+        RqtRoscommUtil._logger.error("list_files: not implemented in ROS2))")
```

Comment:
Please create a ticket for this to follow up on.

Reply:
https://github.com/ros-visualization/rqt/issues/154